build.gradle.kts

```diff
@@ -133,6 +133,7 @@ tasks.withType<KotlinCompile> {
 tasks.withType<Test> {
     dependsOn(tasks.withType<GenerateTask>())
     useJUnitPlatform()
+    jvmArgs("--add-opens", "java.base/java.time=ALL-UNNAMED")
```

Comment:
Why was this necessary?

Reply:
Sorry, @Neil-Massey, I forgot to add the explanation for this. 
Without this, we get an error when we want to spy `java.time` objects:

<img width="891" alt="image" src="https://user-images.githubusercontent.com/39367710/204880118-d36b20ac-a418-4c28-9895-21478ac65f29.png">

I needed to spy a `Clock` instance in the `PrintRequestMapperTest` to use `@InjectMocks` instead of using reflection to set the dependencies in the `PrintRequestMapper` hence added the above change.

Reply:
Thanks @abdullah-sanver-valtech - I wondered if that may have bee the reason but wasn't sure 👍